src/components/TransactionsList/TransactionsList-container.tsx

```diff
@@ -42,15 +37,12 @@ const filtersName: HeaderColsInterface[] = [
   { id: 'amountMax', numeric: false, disablePadding: false, label: 'Amount max' }
 ];
 
-type Order = 'asc' | 'desc';
-type OrderBy = string;
-
-const mapState = (state: any): any => ({
+const mapState = (state: any): { blokchain: Array<Block> } => ({
   blokchain: getBlockchainByDatasource(state, state.dataSource)
 });
 
-let initState: any[] = [];
-const filtersOptions: {[key: string]: string} = {};
+let initState: Array<Block> = [];
```

Comment:
To się odnosi do tego, co napisałm poniżej Damian. Tak lintowi taka forma zapisu się nie spodoba :D Propozycja Damiana powinna być OK.